### PR TITLE
Properly fix displaying virtual session provider in admin panel

### LINKED ISSUE #7147: SessionProvider MySQL credentials are shown in the admin GUI as plaintext
- Gitea version (or commit ref): 1.8.1
- Git version: 2.7.4
- Operating system: Ubuntu 16.04
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description
Greetings!
I'm using Gitea 1.8.1 with MySQL 5.7. And if I'm using MySQL for session storing purposes, I can see the credentials in GUI as plaintext. 

**Steps to reproduce**

1. Select MySQL like a sessions storage in the config file:
```
[session]
PROVIDER        = mysql
PROVIDER_CONFIG = someclient:somepassword@tcp(srv-mysql:3306)/someclient
```
2. Reload Gitea
3. Login in into Gitea with admin credentials
4. Follow this way: Site Administration ⇒ Configuration ⇒ Session Configuration ⇒ Provider Config

### PR DESCRIPTION
**Pull request by zeripath**
Although #7300 properly shadows the password from the virtual session
provider, the template displaying the provider config still presumed
that the config was JSON.

This PR updates the template and properly hides the Virtual Session
provider.

Fixes #7147

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

**Comment by guillep2k** [comment]
I think the referenced issue `Fixes #7127` in the description and in the commit messages is incorrect?

**Comment by zeripath** [comment]
Yup. Don't know what's wrong with my brain sometimes!

**Review by axifive — APPROVED** [review]

File: routers/admin/admin.go [hunk 1]
```diff
@@ -314,8 +308,14 @@ func Config(ctx *context.Context) {
 	ctx.Data["CacheItemTTL"] = setting.CacheService.TTL
 
 	sessionCfg := setting.SessionConfig
+	if sessionCfg.Provider == "VirtualSession" {
+		var realSession session.Options
+		if err := json.Unmarshal([]byte(sessionCfg.ProviderConfig), &realSession); err != nil {
+			log.Error("Unable unmarshall session config for virtualed provider config: %s\nError: %v", sessionCfg.ProviderConfig, err)
```
Comment (silverwind) [inline routers/admin/admin.go]:
```suggestion
			log.Error("Unable to unmarshal session config for virtualed provider config: %s\nError: %v", sessionCfg.ProviderConfig, err)
```

**Comment by techknowlogick** [comment]
@zeripath please send backport :)

**Pull request merged**